http/oidc/src/main/java/org/wildfly/security/http/oidc/AuthenticatedActionsHandler.java

```diff
@@ -1,6 +1,6 @@
 /*
  * JBoss, Home of Professional Open Source.
- * Copyright 2021 Red Hat, Inc., and individual contributors
+ * Copyright 2024 Red Hat, Inc., and individual contributors
```

Comment:
Just a minor comment, we don't need to update the year on existing files.

Reply:
original date returned